x-pack/test/functional/services/aiops/explain_log_rate_spikes_analysis_table.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProvidedType } from '@kbn/test';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+type ExpectedSectionTableEntries = Record<string, string>;
+export interface ExpectedSectionTable {
+  section: string;
+  expectedEntries: ExpectedSectionTableEntries;
+}
+
+export type AnalyticsTableRowDetails = Record<'jobDetails', ExpectedSectionTable[]>;
+
+export type ExplainLogRateSpikesAnalysisTable = ProvidedType<
```

Comment:
At this point, we don't use this service in other services, so we don't really need to export this type. It's doesn't hurt to have it here though, so I'll leave it to you whether or not to remove this bit.

Reply:
Thanks for catching, was a leftover from copy+pasting, removed all unused types in [1f9f454](https://github.com/elastic/kibana/pull/138661/commits/1f9f45402238398da7a4830013a4bb6ba09a5950).